doc/engine/Calculation-Workflow.md

```diff
@@ -0,0 +1,37 @@
+### Calculation Workflow
+
+Calculation workflows are broken into 6 distinct phases. In order of execution, they are: pre-execute, execute, post-execute, post-process, export, and cleanup. The purpose of each phase is defined below.
+
+### Pre-execute
+
+The pre-execution phase in intended for:
+- parsing and preparing inputs (“unzipping” input XML files into the database, for example)
+- initialising the calculation “work packages” (and saving them to the database)
+- initialising “container” DB records to hold computed results
+
+### Execute
+
+The execution phase is where the bulk of the calculation work happens. During this phase, the control node manages task queuing and waits all of the workers to finish the calculation. In this phase the majority of the control node’s time is spent waiting for “completed task” signals from the worker, which indicates that a new task can be enqueued. When there are no more tasks left to enqueue, the control node will simply wait for all of the tasks to finish. (By convention, all calculation results are saved directly to the database by the workers.)
+

```

Comment:
Remove "By convention, all calculation results are saved directly to the database by the workers."